catalog/USGS/USGS_NLCD_RELEASES_2021_REL_TCC_v2021-4.jsonnet

```diff
@@ -0,0 +1,392 @@
+local ee_const = import 'earthengine_const.libsonnet'; 
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local id = 'USGS/NLCD_RELEASES/2021_REL/TCC/v2021-4';
+local version = 'v2021-4';
+local subdir = 'USGS';
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local license = spdx.proprietary;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'USFS Tree Canopy Cover ' + version + ' ' + '(CONUS and OCONUS)',
+  'gee:skip_indexing': true,
+  version: version,
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    This product is part of the Tree Canopy Cover (TCC) data suite. It includes modeled TCC, standard error (SE), and 
+    National Land Cover Database's (NLCD) TCC data for each year. TCC data produced by the the United States Department 
+    of Agriculture, Forest Service (USFS) are included in the Multi-Resolution Land Characteristics (MRLC) consortium 
+    that is part of the National Land Cover Database (NLCD) project managed by the United States (US) Geological Survey (USGS).
+
+    The Science TCC product and NLCD TCC are remote sensing-based map output produced by the USFS. The objecive of TCC Science 
+    and NLCD TCC are to develop a consistent approach using the latest technology and advancements in TCC mapping to produce a 
+    "best available" map of TCC across the Conterminous United States (CONUS) and southeast Alaska, Hawaii and 
+    Puerto Rico-US Virgin Islands (OCONUS).
+
+    Model outputs include Science TCC, Science SE and NLCD TCC. Science TCC and SE include data from 2008 through 2021. 
+    NLCD TCC include data from 2011 through 2021, with data fully masked in 2008, 2009 and 2010. 
+
+    *Science TCC is the raw direct model outputs.
```

Comment:
Please add a space after asterisks to create valid markdown lists.

You can tools like https://markdownlivepreview.com/ to check markdown syntax, though they are not quite so easy to use with these files, as the description text has a four-space whitespace in the middle that messes them up.

You can use importstr (example: https://github.com/google/earthengine-catalog/blob/main/catalog/JRC/catalog.jsonnet#L16) if you want to work with more manageable markdown files, but we'll double-check formatting before publishing anyway.

Reply:
Added the space. Thanks for the links to the markdown checkers!